packages/core/index.ts

```diff
@@ -58,6 +61,9 @@ export type { EventsControllerState } from './src/controllers/EventsController.j
 export { TransactionsController } from './src/controllers/TransactionsController.js'
 export type { TransactionsControllerState } from './src/controllers/TransactionsController.js'
 
+export { PortfolioTokenController } from './src/controllers/PortfolioTokenController.js'
```

Comment:
This is from another feature we started on hackweek but declined. So we could remove `PortfolioTokenController`